app/models/follow.rb

```diff
@@ -63,4 +65,10 @@ def decrement_cache_counters
     account&.decrement_count!(:following_count)
     target_account&.decrement_count!(:followers_count)
   end
+
+  def invalidate_hash_cache
+    return if account&.local? && target_account&.local?
```

Comment:
I don't think they can be nil when this is executed. If they could, you'd run into a nil error in the line below.